selector.go

```diff
@@ -72,10 +85,11 @@ func NewIDSelectorNot(exclude []int64) (*IDSelectorNot, error) {
 	var sel *C.FaissIDSelectorNot
 	if c := C.faiss_IDSelectorNot_new(
 		&sel,
-		batchSelector.sel,
+		batchSelector.GetFaissSelector(),
 	); c != 0 {
 		batchSelector.Delete()
 		return nil, getLastError()
 	}
-	return &IDSelectorNot{sel: (*C.FaissIDSelector)(sel), batchSel: batchSelector.sel}, nil
+	return &IDSelectorNot{sel: (*C.FaissIDSelector)(sel),
+		batchSel: batchSelector.GetFaissSelector()}, nil
```

Comment:
Formatting looks funky, would you change to ..
```
                return &IDSelectorNot{
                        sel:      (*C.FaissIDSelector)(sel),
                        batchSel: batchSelector.Get(),
                }, nil
```

Reply:
cool, just did it so we don't go past the 80 char limit.